### ISSUE TITLE
Source bin/agent looking for SOCK_FILE and PID_FILE in wrong location

### ISSUE THREAD

**Issue by voxxit** [DataDog/dd-agent#1863]
Last night, I installed from source, and I found that `bin/agent` script was looking for `$SOCK_FILE` and `$PID_FILE` in the `supervisord/` directory, whereas supervisor itself seems to want to put the files in the `/../run/` directory (just above it).

Info about my build:

```
Collector (v 5.4.4)
===================

  Status date: 2015-08-26 14:02:01 (16s ago)
  ...
  Platform: Linux-2.6.32-amazon-xen-r8-x86_64-Intel-R-_Xeon-R-_CPU_E5-2650_0_@_2.00GHz-with-gentoo-1.12.11.1
  Python Version: 2.7.10
```

The fix is quite simple - change L9-10 of `bin/agent` to the following:

```bash
SOCK_FILE='run/agent-supervisor.sock'
PID_FILE='run/supervisord.pid'
```

Am I doing something wrong when I install, or is this bug report accurate? Thanks!

**Comment by remh** [comment]
Thanks a lot for thefeedback @voxxit !

@elafarge can you review this issue please ?

**Comment by remh** [comment]
Thanks a lot @voxxit you're absolutely right. This will be fixed by #1913
